Makefile

```diff
@@ -49,7 +49,7 @@ lint: lint-flake8 lint-pylint
 # E501 and F401 are ignored because Pylint performs similar checks.
 # W504 ignored since it requires line breaks after binary operators.
 lint-flake8:
-	flake8 . --ignore E501,F401,W504 --exclude docs/_build
+	flake8 . --ignore E501,F401,W504 --exclude docs/_build,.eggs
```

Comment:
Python 3.7 is creating a `/.eggs` directory for --editable installations and we do not want to lint that code.

Reply:
What about we created a dedicated PR for this fix, since it is a different concern?